server/src/main/resources/org/elasticsearch/common/reference-docs-links.json

```diff
@@ -43,5 +43,6 @@
   "MAX_SHARDS_PER_NODE": "size-your-shards.html#troubleshooting-max-shards-open",
   "FLOOD_STAGE_WATERMARK": "fix-watermark-errors.html",
   "X_OPAQUE_ID": "api-conventions.html#x-opaque-id",
-  "FORMING_SINGLE_NODE_CLUSTERS": "modules-discovery-bootstrap-cluster.html#modules-discovery-bootstrap-cluster-joining"
+  "FORMING_SINGLE_NODE_CLUSTERS": "modules-discovery-bootstrap-cluster.html#modules-discovery-bootstrap-cluster-joining",
+  "ALLOCATION_EXPLAIN_MAX_RETRY": "cluster-allocation-explain.html#_maximum_number_of_retries_exceeded"
```

Comment:
Please add a fixed `[[anchor-name]]` in the docs rather than using the `#_auto_generated` one which might change inadvertently.

Reply:
See https://github.com/elastic/elasticsearch/pull/113667 which forbids this.

Reply:
[X-post example](https://github.com/elastic/elasticsearch/pull/113656#discussion_r1778933646).